faunadb-scala/src/main/scala/faunadb/values/Value.scala

```diff
@@ -102,6 +102,8 @@ object Value {
 
   /** Create a date value. */
   def apply(localdate: LocalDate): Value = DateV(localdate)
+
+  implicit def from[T](obj: T)(implicit encoder: Encoder[T]): Value = encoder.encode(obj)
```

Comment:
I was thinking about this function.

I agree with its purpose but, I don't agree with its place. See, the `Encoder` knows `Value` hierarchy, and with this method here, the Value knows about encoders. Also, the top level companion object of `Value` trait, can create instances of any of their children using the encoder. I feel like this is becoming ciclic.

I would rather place this in the `Codec` object or in the `Encoder` companion object. Just so we keep all the knowledge about decode/encode in the same place.